build.sh

```diff
@@ -76,6 +76,42 @@ function clean_image {
   fi
 }
 
+# Pull image based on FROM, before we build our own.
+function pull_image {
+  local dockerfile="$1"
+
+  # Get image_name from Dockerfile before pulling.
+  while read -r line; do
+    if ! grep -q "^FROM" <<< "$line"; then
+      continue
+    fi
+
+    image_name=$(echo "$line" | cut -d ' ' -f2)
+    # In case FROM scratch is defined, skip it
+    if grep -q "scratch" <<< "$image_name"; then
+        continue
+    fi
+    # In case FROM ubi8 as ubi-base is mentioned
+    # pull ubi8 image
+    if grep -q " as " <<< "$line"; then
+        image_name=$(echo "$line" | cut -d ' ' -f2)
+    fi
```

Comment:
I do not see a difference to `image_name=$(echo "$line" | cut -d ' ' -f2)` command slightlz above this part.

Reply:
Good catch, It seems, like I was trying to get third argument, but second is right. I will remote it.